GrandCentralBoard/Widgets/Harvest/FetchAccessTokenRequest.swift

```diff
@@ -0,0 +1,55 @@
+//
+//  FetchAccessTokenRequest.swift
+//  GrandCentralBoard
+//
+//  Created by Karol Kozub on 2016-04-14.
+//  Copyright © 2016 Oktawian Chojnacki. All rights reserved.
+//
+
+import Foundation
+import GrandCentralBoardCore
+
+
+class FetchAccessTokenRequest {
```

Comment:
I am not sure about the name, I would use sth like `AccessTokenProvider`. What do you think?